packages/dawn4py/package.py

```diff
@@ -30,3 +31,6 @@ class Dawn4py(PythonPackage):
 
     # will test that dawn4py is importable after the install
     import_modules = ['dawn4py']
+
+    def setup_build_environment(self, spack_env, run_env):
+        spack_env.set('LLVM_ROOT', self.spec['llvm'].prefix)
```

Comment:
```suggestion
    def setup_build_environment(self, env):
        env.set('LLVM_ROOT', self.spec['llvm'].prefix)
```

Ah, `setup_build_environment` takes only one argument. My bad